app/views/families/_form.html.arb

```diff
@@ -29,6 +29,7 @@ context.instance_exec do
 
   inputs t('.precheck') do
     input :precheck_status, as: :select, include_blank: false, collection: precheck_statuses_select_collection
+    input :required_team_action, as: :check_boxes, collection: Family::REQUIRED_ACTION_TEAMS, label: t('.action_required')
```

Comment:
FYI instead of specifying the label it's possible to set a translation for attribute name, which will then be used wherever a label is needed: https://github.com/CruGlobal/staff-conf-onsite/blob/f64db928379a5dcbe167960cb8093106e319f4d8/config/locales/en.yml#L169-L176

Reply:
Thanks! I knew I was doing this one wrong ;)